### ISSUE TITLE
Finetuning Struck

### ISSUE THREAD

**Issue by Subbu-ssr** [ASLP-lab/DiffRhythm#92]
(fine) ubuntu@104-171-202-71:~/Subbu_Finetuning/DiffRhythm/scripts$ bash train.sh 
Loaded pretrained model from /home/ubuntu/Subbu_Diffrhythm/DiffRhythm/pretrained/models--ASLP-lab--DiffRhythm-full/snapshots/8971c6a2709d6619e9865ce8f81ee24c06ef788c/cfm_model.pt
Original model parameters: 1109315648
Trainable parameters: 12348416
LoRA parameter efficiency: 1.11%
Namespace(config_file='defaults.ini', wandb_config=None, exp_name='beatles_finetune', batch_size=1, max_frames=2048, min_frames=512, num_workers=4, seed=42, accum_batches=1, checkpoint_every=1000, ckpt_path='', pretrained_ckpt_path='', pretransform_ckpt_path='', model_config='/home/ubuntu/Subbu_Finetuning/DiffRhythm/config/diffrhythm-1b.json', model_path='/home/ubuntu/Subbu_Diffrhythm/DiffRhythm/pretrained/models--ASLP-lab--DiffRhythm-full/snapshots/8971c6a2709d6619e9865ce8f81ee24c06ef788c/cfm_model.pt', dataset_config='', save_dir='/home/ubuntu/Subbu_Finetuning/DiffRhythm/ckpts/beatles_finetune', max_grad_norm=1.0, grad_accumulation_steps=1, learning_rate=0.0001, epochs=5, num_warmup_updates=1000, save_per_updates=1000, last_per_steps=1000, file_path='/home/ubuntu/Subbu_timestamps/train.scp', audio_drop_prob=0.1, cond_drop_prob=0.1, style_drop_prob=0.1, lrc_drop_prob=0.1, use_style_prompt=1, reset_lr=0, resumable_with_seed=42, downsample_rate=2048, grad_ckpt=0, sampling_rate=44100)
Epoch 1/5:   0%|                                                         | 0/47 [00:00<?, ?step/s]

Hi everyone,

Can anyone help me to figure out this iam strucked here it is not showing any progress it stopped here for the 30 mins ,

Please help me to solve this issue

Thanks in advance

**Comment by NZqian** [comment]
What is your GPU and is its utilization normal?

**Comment by Subbu-ssr** [comment]
Its A6000 GPU and the respective process allocated 8690 gpu vram was allocated and running fine .


Thanks for your reply

**Comment by lidongming1124** [comment]
数据集和原始不一致，虽然格式都是pt， 项目自带的就可以很快训练。

**Comment by Subbu-ssr** [comment]
i also want to know the dataset preparation process i have tried and extracted the features as diffrhythm model expects (sample dataset .pt files are provided in the folder dataset) with the same shapes as they provided for both latent,style and lrc also but i didn't acheived the best model.pt file from finetuning it and the output was not good so much noise in it from finetuned model.pt file

please clarify me or provide me some codes to finetune it on my songs


Thanks for your reply @NZqian

**Comment by Subbu-ssr** [comment]
Hi @NZqian  I completed the finetuning of diffrhythm i got model.pt files i tried to play all of them but i got only noisy outputs.
But the i can hear singing in the background but noise is very high in that output can you please suggest me some tips to adjust and make me to get a perfect output from my finetuned model

Thank you @NZqian
